### PR TITLE
V1 timetracking

### PR DESCRIPTION
**Pull request by maartenscholz**

### CODE REVIEW

File: src/apiary.apib [hunk 1]
```diff
@@ -17,6 +17,8 @@ This is the Teamleader API v2 documentation.
 
 :[Files](files.apib)
 
-:[Meetings](meetings.apib)
+:[Meetings](meetings.apib
```
Comment (steftrenson) [inline src/apiary.apib]: Missing )

File: src/timetracking.apib [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+### getTimetrackingForTask.php [POST /getTimetrackingForTask.php]
+
++ Request (application/json)
+
+    + Attributes
+        + task_id: 1 (number)
+
++ Response 200 (application/json)
+
+    + Attributes
```
Comment (fowbi) [inline src/timetracking.apib]: Missing backticks for the attribute examples (used in previous examples)

File: src/timetracking.apib [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        + company_name: `Doe inc.` (string)
+        + contact_name: `John Doe` (string)
+        + hourly_cost: `10` (number)
+
+### getTimetrackingForTask.php [POST /getTimetrackingForTask.php]
+
++ Request (application/json)
+
+    + Attributes
+        + task_id: 1 (number)
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks in example

File: src/timetracking.apib [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+                + 00
+                + 06
+                + 12
+                + 21
+                + CM
+                + EX
+                + MC
+                + VCMD
+        + product_id_1: `1` (number, optional) - Id of the product for the additional cost
+        + subtitle_1: subtitle (string, optional) - Additional cost subtitle
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks for the example value

File: src/timetracking.apib [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+        + subtract_break: `1` (enum[number], optional) - If this parameter is set to 1 lunchbreaks will be substracted from the timetracking
+             + Members
+                 + 0
+                 + 1
+        + related_object_type: task (enum[string], optional) - What kind of item this activity is related to
+            + Members
+                + none
+                + task
+        + related_object_id: `1` (number, optional) - If this activity is related to a task, pass in the task ID
+        + description_1: description (string, optional) - Additional costs description
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks for the example value

File: src/timetracking.apib [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+    additional cost.
+
+    + Attributes
+        + timetracking_id: `1` (number) - ID of the timetracking to update
+        + description: description (string, optional) - Description of the activity
+        + start_date: `1498048402` (number, optional) - Start date of the activity as Unix timestamp (seconds)
+        + end_date: `1498048402` (number, optional) - End date of the activity as Unix timestamp (seconds)
+        + worker_id: `12` (number, optional) - Id of the user
+        + task_type_id: `13` (number, optional) - Type id of the task
+        + for: company (enum[string], optional) - Who this activity should be billed to
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks for the example value

File: src/timetracking.apib [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+        + for_id: `1` (number, optional) - The id of who this activity should be billed to
+        + invoiceable: `0` (enum[number], optional) - Can this time be invoiced later or not? This parameter has no effect for project-related timetracking, because invoiceability is determined by the related milestone.
+            + Members
+                + 0
+                + 1
+        + subtract_break: `1` (enum[number], optional) - If this parameter is set to 1 lunchbreaks will be substracted from the timetracking
+             + Members
+                 + 0
+                 + 1
+        + related_object_type: task (enum[string], optional) - What kind of item this activity is related to
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks for the example value(s)

File: src/timetracking.apib [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
++ Request (application/json)
+
+    Additional cost can be added with optional params. These params are suffixed with the index (1 based) of the
+    additional cost  (`<param>_<index_of_additional_cost>`).
+    So the description of the first additional cost will be `description_1` and `description_2` for the second
+    additional cost.
+
+    + Attributes
+        + timetracking_id: `1` (number) - ID of the timetracking to update
+        + description: description (string, optional) - Description of the activity
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks for the example

File: src/timetracking.apib [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+                + 00
+                + 06
+                + 12
+                + 21
+                + CM
+                + EX
+                + MC
+                + VCMD
+        + product_id_1: `1` (number, optional) - Id of the product for the additional cost
+        + subtitle_1: subtitle (string, optional) - Additional cost subtitle
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks in the example value

File: src/timetracking.apib [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+        + subtract_break: `1` (enum[number], optional) - If this parameter is set to 1 lunchbreaks will be substracted from the timetracking
+             + Members
+                 + 0
+                 + 1
+        + related_object_type: task (enum[string], optional) - What kind of item this activity is related to
+            + Members
+                + none
+                + task
+        + related_object_id: `1` (number, optional) - If this activity is related to a task, pass in the task ID
+        + description_1: description (string, optional) - Additional costs description
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks

File: src/timetracking.apib [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+        + for_id: `1` (number, optional) - The id of who this activity should be billed to
+        + invoiceable: `0` (enum[number], optional) - Can this time be invoiced later or not? This parameter has no effect for project-related timetracking, because invoiceability is determined by the related milestone.
+            + Members
+                + 0
+                + 1
+        + subtract_break: `1` (enum[number], optional) - If this parameter is set to 1 lunchbreaks will be substracted from the timetracking
+             + Members
+                 + 0
+                 + 1
+        + related_object_type: task (enum[string], optional) - What kind of item this activity is related to
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks for the example value(s)

File: src/timetracking.apib [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+    So the description of the first additional cost will be `description_1` and `description_2` for the second
+    additional cost.
+
+    + Attributes
+        + description: description (string) - Description of the activity
+        + start_date: `1498048402` (number) - Start date of the activity as Unix timestamp (seconds)
+        + end_date: `1498048402` (number) - End date of the activity as Unix timestamp (seconds)
+        + worker_id: `12` (number) - Id of the user
+        + task_type_id: `13` (number) - Type id of the task
+        + for: company (enum[string], optional) - Who this activity should be billed to
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks for the example value(s)

File: src/timetracking.apib [hunk 13]
```diff
@@ -0,0 +1,208 @@
...
+
++ Request (application/json)
+
+    Additional cost can be added with optional params. These params are suffixed with the index (1 based) of the
+    additional cost  (`<param>_<index_of_additional_cost>`).
+    So the description of the first additional cost will be `description_1` and `description_2` for the second
+    additional cost.
+
+    + Attributes
+        + description: description (string) - Description of the activity
```
Comment (fowbi) [inline src/timetracking.apib]: missing backticks for the example

**Comment by maartenscholz** [comment]
Fixed conflicts and added backticks.
The backticks are actually almost never needed (you saw that stopped using them in later endpoints). But I think everyone is using them so added them.

**Pull request merged**